### DISCUSSION TITLE
Memory.dll in python

### DISCUSSION THREAD

**Discussion by Furkan125** [erfg12/memory.dll#118] [Q&A] [answered]
Hi, its possible to use Memory.dll in Python ?

**Comment by erfg12** [comment, marked as answer]
I don't really know Python. I've built a few programs with it, but I don't know more than that. You could follow this guide if you want to make a trainer with Python: https://guidedhacking.com/threads/guide-to-learning-python-game-hacking.13506/

**Reply by Furkan125** [reply]
Thanks for your answer. It's very helpful.
